### PR TITLE
refactor(styles): Use border box and clean up simple tab copy/pasta

### PR DESCRIPTION
**Pull request by ckniffen**
### Context of Change

`box-sizing: border-box` should be used in all modern sites to make
responsive styles much simpler.  This required changes to hardcoded
widths on the homepage and simple tab pages to increase the width by the
padding and borders otherwise the new width would be smaller.

As part of this work I noticed that the code between
`Validators/simpleTabs.scss` and `Transactions/simpleTab.scss` was
nearly identical.  The similarities are now normalized into
`shared/css/simpleTab.scss`.

This conflicts with #218

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### CODE REVIEW

File: src/containers/Validators/simpleTab.scss [hunk 1]
```diff
@@ -1,141 +1,21 @@
...
-        text-align: right;
-        vertical-align: middle;
-
         img {
           height: 16px;
         }
-
-        @include medium;
-
-        .currency {
```
Comment (ckniffen) [inline src/containers/Validators/simpleTab.scss:76]: Line 76 - 127 do not apply to validators and must have been copied originally from `Transactions/simpleTab.scss`

File: src/containers/Validators/validator.scss [hunk 2]
```diff
@@ -149,21 +149,5 @@
     @include for-size(big-desktop-up) {
       width: 1140px;
     }
-
-    .react-json-view {
```
Comment (ckniffen) [inline src/containers/Validators/validator.scss:153]: json viewer never appears on validators pages.

File: src/containers/Transactions/simpleTab.scss [hunk 3]
```diff
@@ -148,38 +86,4 @@
       }
     }
   }
-
-  .index {
```
Comment (ckniffen) [inline src/containers/Transactions/simpleTab.scss:152]: Moved to `shared/css/simpleTab.scss`

File: src/containers/Transactions/simpleTab.scss [hunk 4]
```diff
@@ -29,47 +18,7 @@
     }
 
     .row {
-      display: flex;
```
Comment (ckniffen) [inline src/containers/Transactions/simpleTab.scss:32]: Moved to `shared/css/simpleTab.scss`

File: src/containers/Validators/simpleTab.scss [hunk 5]
```diff
@@ -153,35 +33,14 @@
   }
 
   .index {
-    width: 275px;
```
Comment (ckniffen) [inline src/containers/Validators/simpleTab.scss:156]: Moved to `shared/css/simpleTab.scss`

**Review by intelliot — APPROVED** [review]

File: src/containers/Transactions/transaction.scss [hunk 6]
```diff
@@ -25,17 +25,17 @@
     margin: 8px auto 0px;
 
     @include for-size(tablet-landscape-up) {
```
Comment (mvadari) [inline src/containers/Transactions/transaction.scss:27]: Why were these values changed?

Comment (ckniffen) [reply src/containers/Transactions/transaction.scss:27]: Anywhere that had a hardcoded width had to be adjusted to include the padding and borders.

**Review by mvadari — APPROVED** [review]

**Pull request merged**